forwarder/src/lib.rs

```diff
@@ -37,10 +37,25 @@ impl AuroraForwarder {
         target_network: AccountId,
         fees_contract_id: AccountId,
     ) -> Self {
+        assert!(
+            !env::state_exists(),
+            "State of the contract is already exist"
+        );
+
+        let target_address = target_address.trim_start_matches("0x").to_string();
+
+        assert!(
+            is_valid_account_id(
+                &env::current_account_id(),
```

Comment:
Performance nit: could avoid calling `env::current_account_id()` multiple times by capturing it as a variable and re-using it both here and in the add full access key promise creation.